estate/models/estate_property.py

```diff
@@ -0,0 +1,151 @@
+from datetime import datetime, timedelta
+from odoo import fields, models, Command
+from odoo import api
+from odoo import exceptions
+from odoo.exceptions import ValidationError
+from odoo.exceptions import UserError
+from odoo.tools.float_utils import float_compare, float_is_zero
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = "ESTATE Properties"
+    _order = "id desc"
+
+    name = fields.Char(string="Title", required=True, help="Property name")
+    description = fields.Text(string="Description", help="Property description")
+    postcode = fields.Text(string="Postcode", help="Property Post code")
+    date_availability = fields.Date(string="Available From",
+                                    default=lambda self: (datetime.now() + timedelta(days=90)).strftime('%Y-%m-%d'),
+                                    copy=False, help="Property Date Availability")
```

Comment:
podrias hacer esta parte tantito mas legible
```suggestion
    date_availability = fields.Date(
       string="Available From",
       default=lambda self: (datetime.now() + timedelta(days=90)).strftime('%Y-%m-%d'),
       copy=False, help="Property Date Availability"
    )
```